src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,67 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Calculator {
+	public String[] splitStr(String calStr){
```

Comment:
```suggestion
	public String[] splitStr(String calStr) {
```

전반적으로 띄어쓰기가 없는데, 코딩 스타일로 정의되어 있는건가요?